### PR TITLE
DOC: Fix warnings related to displayed math expressions

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Mark as raw strings docstrings using LaTeX math expressions
- DOC: Fix warnings related to displayed math expressions

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Failures are unrelated.

File: dipy/align/imaffine.py [hunk 1]
```diff
@@ -903,7 +903,7 @@ def __init__(
         options=None,
         verbosity=VerbosityLevels.STATUS,
     ):
-        """Initialize an instance of the AffineRegistration class.
+        r"""Initialize an instance of the AffineRegistration class.
```
Comment (skoudoro) [inline dipy/align/imaffine.py:906]: I do not understand why you need a raw docstring here. Can you explain ?

File: dipy/data/fetcher.py [hunk 2]
```diff
@@ -1691,7 +1691,7 @@ def get_fnames(*, name="small_64D"):
 
 
 def read_qtdMRI_test_retest_2subjects():
-    """Load test-retest qt-dMRI acquisitions of two C57Bl6 mice. These
+    r"""Load test-retest qt-dMRI acquisitions of two C57Bl6 mice. These
```
Comment (skoudoro) [inline dipy/data/fetcher.py:1694]: same here

File: dipy/reconst/fwdti.py [hunk 3]
```diff
@@ -239,7 +239,7 @@ def predict(self, gtab, S0=1):
 def wls_iter(
     design_matrix, sig, S0, Diso=3e-3, mdreg=2.7e-3, min_signal=1.0e-6, piterations=3
 ):
-    """Applies weighted linear least squares fit of the water free elimination
+    r"""Applies weighted linear least squares fit of the water free elimination
```
Comment (skoudoro) [inline dipy/reconst/fwdti.py:242]: same here

File: dipy/reconst/fwdti.py [hunk 4]
```diff
@@ -729,7 +729,7 @@ def nls_fit_tensor(
     weighting=None,
     sigma=None,
 ):
-    """
+    r"""
```
Comment (skoudoro) [inline dipy/reconst/fwdti.py:732]: same here

**Review by skoudoro — COMMENTED** [review]
hi @jhlegarreta,

Not sure why you not a raw doctring when there is NOT a backslash. 

I believe, the raw string is needed only for backslashes conflict. Please keep me updated.  I will also look to clarify this information. 

Thank you

File: dipy/align/imaffine.py (hunk 1 above)
Comment (jhlegarreta) [reply dipy/align/imaffine.py]: My bad. I may have added it thinking that a warning in this method was raised by the math expressions. Removed in 1b8fea5.

File: dipy/data/fetcher.py (hunk 2 above)
Comment (jhlegarreta) [reply dipy/data/fetcher.py:1694]: This one has a backslash in
https://github.com/dipy/dipy/pull/3325/files#diff-674b7eaaf375ba4af888af6d75137f735bb8608d56d3b9c84d4d8f421b397282R1696

so leaving it.

File: dipy/reconst/fwdti.py (hunk 3 above)
Comment (jhlegarreta) [reply dipy/reconst/fwdti.py]: My bad. I may have added it thinking that a warning in this method was raised by the math expressions. Removed in 1b8fea5.

**Review by skoudoro — APPROVED** [review]
All good, Thank you for the update @jhlegarreta, merging

File: dipy/reconst/fwdti.py (hunk 4 above)
Comment (jhlegarreta) [reply dipy/reconst/fwdti.py]: My bad. I may have added it thinking that a warning in this method was raised by some math expression, but no backslash is present in this method. Removed in 1b8fea5.

**Pull request merged**